group_vars/location_vaterhaus/snmp.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+snmp_devices:
+  - hostname: vaterhaus-zwingli
+    address: 10.230.192.202
+    snmp_profile: af60
+
+snmp_devices:
```

Comment:
this will not append but override the list. Please remove that line and you should be good to go